crates/cxx-qt-gen/src/generator/rust/signals.rs

```diff
@@ -213,7 +221,9 @@ pub fn generate_rust_signal(
                 span =>
                 #(#cfgs)*
                 impl cxx_qt::signalhandler::CxxQtSignalHandlerClosure for #closure_struct {
+                    #(#cfgs)*
                     type Id = cxx::type_id!(#signal_handler_alias_namespaced_str);
+                    #(#cfgs)*
                     type FnType = dyn FnMut(#self_type_qualified, #(#parameters_qualified_type),*) + Send;
```

Comment:
actually i think we don't need these cfgs they should be picked up by the impl cfg?